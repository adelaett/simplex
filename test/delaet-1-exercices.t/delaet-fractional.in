6
1
25 18 10 12 45 14
60
20 16 7 10 30 12