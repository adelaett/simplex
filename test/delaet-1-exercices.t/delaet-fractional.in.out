maximize   25  18  10  12  45  14  0  |  0
--------------------------------------------------------------------------------
subject to 20  16  7  10  30  12  1  |  60
entering : 0
leaving :  6
maximize   0  -2  5/4  -1/2  15/2  -1  -5/4  |  -75
--------------------------------------------------------------------------------
subject to 1  4/5  7/20  1/2  3/2  3/5  1/20  |  3

entering : 2
leaving :  0
maximize   -25/7  -34/7  0  -16/7  15/7  -22/7  -10/7  |  -600/7
--------------------------------------------------------------------------------
subject to 20/7  16/7  1  10/7  30/7  12/7  1/7  |  60/7

entering : 4
leaving :  2
maximize   -5  -6  -1/2  -3  0  -4  -3/2  |  -90
--------------------------------------------------------------------------------
subject to 2/3  8/15  7/30  1/3  1  2/5  1/30  |  2

The problem is FEASIBLE and BOUNDED
One solution is x = { 0, 0, 0, 0, 2, 0 }
The objective value for this solution is: 90
The number of pivots is: 3
The rule used: bland
